packages/lib/src/vis/line/DataCurve.tsx

```diff
@@ -33,6 +31,9 @@ interface Props {
   curveType?: CurveType;
   visible?: boolean;
   onLineClick?: (index: number, evt: ThreeEvent<MouseEvent>) => void;
+  onLinePointerEnter?: (index: number, evt: ThreeEvent<PointerEvent>) => void;
+  onLinePointerOut?: (index: number, evt: ThreeEvent<PointerEvent>) => void;
```

Comment:
This brings the API of `DataCurve` in line with the one of `ScatterPoints`: https://github.com/silx-kit/h5web/pull/1070/files#diff-b2664ce21931d3e6d943701c9a406531d81fde57550a7316700214b18d9434d5R32